src/lib/support/BufferReader.h

```diff
@@ -58,6 +58,18 @@ class Reader
      */
     uint16_t OctetsRead() const { return static_cast<uint16_t>(mReadPtr - mBufStart); }
 
+    /**
+     * Number of octets we have remaining to read.  Can be useful for logging.
+     */
+    uint16_t Remaining() const { return mAvailable; }
```

Comment:
@bzbarsky-apple Does this match what you were about to do ? I recommended to @holbrookt that he prototype his code based on an equivalent incoming change on your part, but I didn't think it would be committed here.

Reply:
#3812 was just merged today. I can rebase before merging this PR so that #3812 is incorporated